### DISCUSSION TITLE
Move datapacks from the world folder to the root folder

### DISCUSSION THREAD

**Discussion by SpiritOTHawk** [PaperMC/Paper#10546] [Ideas]
### Describe alternatives you've considered.

Not known

### Other

`More or less like this`
![image](https://user-images.githubusercontent.com/76490476/180604597-7bae123f-45c7-40f5-aa6e-0cf4ba2e13c3.png)

**Comment by Astralchroma** [comment]
This might be something Paper can't / won't do as a result of maintaining Spigot compatibility, although don't take my word for it.

However it seems that this *might* make some sense considering datapacks are applied globally and not on a per world basis anyway.

**Comment by Machine-Maker** [comment]
I don’t think I’d do this by default, but perhaps a command line flag to change the location. 

https://github.com/PaperMC/Paper/pull/6527
